src/windows/ScrollView.tsx

```diff
@@ -0,0 +1,51 @@
+/**
+* ScrollView.tsx
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* RN Windows-specific implementation of the cross-platform ScrollView abstraction.
+*/
+
+import React = require('react');
+import RN = require('react-native');
+import {ScrollView as ScrollViewBase} from '../native-common/ScrollView';
+
+import EventHelpers from '../native-common/utils/EventHelpers';
+
+export class ScrollView extends ScrollViewBase {
+
+    protected _render(props: RN.ScrollViewProps): JSX.Element {
+
+        var onKeyDownCallback = this.props.onKeyPress ?
+                // We have a callback function, call the wrapper
+                this._onKeyDown :
+                undefined;
```

Comment:
This looks indented a little too much. I think it should be indented 4 spaces relative to the previous line.

Reply:
Ok (wth are the linters good for??)

---
In reply to: [159358840](https://github.com/Microsoft/reactxp/pull/404#discussion_r159358840) [](ancestors = 159358840)